maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -108323,7 +108325,7 @@
 "iDJ" = (
 /obj/machinery/atmospherics/binary/passive_gate{
 	dir = 4;
-	name = "Waste to Space"
+	name = "Waste to Wasting"
```

Comment:
I don't think that's a good name. Same on all other instances.

Reply:
I'm open to suggestion, I could rename it to "Waste to Port".

Reply:
This was after people reporting that Core to Space was "broken", but it doesn't go to Space, it goes to a port which then takes it away, but it starts off, and needs to be set-up.

Reply:
You could use the word Outside and http://www.photographers1.com/Sailing/ShipboardDirections.gif to describe where the pipe leads to.